### PR TITLE
Add property tests for `EntityType`

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

File: grakn-core/src/main/java/ai/grakn/concept/EntityType.java [hunk 1]
```diff
@@ -77,10 +85,12 @@
     EntityType deletePlaysRole(RoleType roleType);
 
     /**
-     * Creates and returns a new Entity instance.
+     * Creates and returns a new Entity instance, whose direct type will be this instance.
```
Comment (aelred) [inline grakn-core/src/main/java/ai/grakn/concept/EntityType.java]: "... will be this *type*" - silly me.

File: grakn-graph/src/test/java/ai/grakn/graph/property/EntityTypePropertyTest.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        assumeThat(type.getRulesOfHypothesis(), empty());
+        assumeThat(type.getRulesOfConclusion(), empty());
+
+        type.delete();
+
+        assertNull(graph.getType(type.getName()));
+    }
+
+    @Property
+    public void whenAddingAnEntityOfAMetaType_Throw(@Meta EntityType type) {
```
Comment (fppt) [inline grakn-graph/src/test/java/ai/grakn/graph/property/EntityTypePropertyTest.java]: Does this take into account the rule meta types `MetaSchema.INFERENCE_RULE` and `MetaSchema.CONSTRAINT_RULE` ? You allowed to add instances to those.

Comment (aelred) [inline grakn-graph/src/test/java/ai/grakn/graph/property/EntityTypePropertyTest.java]: Actually this is just a badly-named test. It only tests for the meta `EntityType` (the only one with an `addEntity()` method).

Comment (fppt) [inline grakn-graph/src/test/java/ai/grakn/graph/property/EntityTypePropertyTest.java]: New request then. Rename test please :)

**Pull request merged**